src/main/java/nl/ordina/jobcrawler/model/assembler/SkillModelAssembler.java

```diff
@@ -0,0 +1,21 @@
+package nl.ordina.jobcrawler.model.assembler;
+
```

Comment:
please add some comment - what is the purpose of the SkillModelAssembler? What is a RepresentationModelAssembler and how is this beneficial to the skill model?